airbyte-integrations/connectors/source-influxdb/source_influxdb/influxdb.yaml

```diff
@@ -0,0 +1,51 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://ap-southeast-2-1.aws.cloud2.influxdata.com"
```

Comment:
I don't think that a URL should be hardcoded given that users should be able to use a self hosted instance of the database as well.

Reply:
Thanks @itaseskii  for pointing this out. 
please let me know whether i can include this in config.json?

Reply:
you should include the url as part of the spec.yml and reference it from there